src/PrestaShopBundle/Resources/config/services/adapter/cms_page_category.yml

```diff
@@ -6,7 +6,7 @@ services:
 
     prestashop.adapter.cms_page_category.command_handler.abstract_cms_page_category_handler:
         class: PrestaShop\PrestaShop\Adapter\CMS\PageCategory\CommandHandler\AbstractCmsPageCategoryHandler
-        abstract: true
+        abstract:  true
```

Comment:
```suggestion
        abstract: true
```

Reply:
this file is not supposed to be there , i'll remove it